.readthedocs.yml

```diff
@@ -1,23 +1,27 @@
-# .readthedocs.yml
 # Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html
 
-# Required
 version: 2
 
 formats:
 - htmlzip
 
 build:
-  os: ubuntu-20.04
+  os: ubuntu-22.04
   tools:
+    # Use most recent version supported by Numba
     python: '3.10'
+  apt_packages:
+  - graphviz
   jobs:
     pre_create_environment:
     - python -m pip install --upgrade setuptools pip
+    post_build:
+    - echo https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#troubleshooting
 
+sphinx:
+  fail_on_warning: true
```

Comment:
This adds the `-W --keep-going` flags to the doc build, which are used in the `build_docs` tox environment.